buildenv/jenkins/README.md

```diff
@@ -274,438 +158,91 @@ Having a complicated regex in the pull request trigger is what allows us to laun
-
-- Test-Sanity-JDK9-linux_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Sanity-JDK9-linux_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Sanity-JDK9-linux_x86-64_cmprssptrs)
-    - Description:
-        - Runs sanity test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Test-Sanity-JDK10-linux_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Sanity-JDK10-linux_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Sanity-JDK10-linux_x86-64_cmprssptrs)
-    - Description:
-        - Runs sanity test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Test-Sanity-JDK8-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Sanity-JDK8-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Sanity-JDK8-win_x86-64_cmprssptrs)
-    - Description:
-        - Runs sanity test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Test-Sanity-JDK9-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Sanity-JDK9-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Sanity-JDK9-win_x86-64_cmprssptrs)
-    - Description:
-        - Runs sanity test suite against the SDK and test material that is passed as parameters
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+Test pipelines for all platforms and versions are available [here](https://ci.eclipse.org/openj9/view/Test/).
 
-- Test-Sanity-JDK10-win_x86-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Test-Sanity-JDK10-win_x86-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Test-Sanity-JDK10-win_x86-64_cmprssptrs)
+- Any platform and version build pipeline:
+    - Name: *Test-`<level>`-JDK`<version>`-`<platform>`*
+        - `<version>` is the number of the supported release, e.g. 8 | 11 | next
+        - `<platform>` is aix_ppc-64_cmprssptrs | linux_390-64_cmprssptrs | linux_ppc-64_cmprssptrs_le | linux_x86-64 | linux_x86-64_cmprssptrs | win_x86-64_cmprssptrs | win_x86
+        - `<level>` 
+            - all platforms except win_x86-64_cmprssptrs and win_x86 is sanity.functional | sanity.system | extended.functional | extended.system
+            - win_x86-64_cmprssptrs and win_x86 is Sanity | Extended
     - Description:
-        - Runs sanity test suite against the SDK and test material that is passed as parameters
+        - Runs full or subsets of the sanity and/or extended test suites against the SDK and test material that is passed as parameters
```

Comment:
I'm not sure what you mean by `subsets`?
What about simply
`Runs the test suite and level indicated against the SDK`?